takahe/settings.py

```diff
@@ -0,0 +1,242 @@
+import secrets
+import urllib.parse
+from pathlib import Path
+from typing import List, Literal, Optional, Union
+
+import dj_database_url
+import sentry_sdk
+from pydantic import AnyUrl, BaseSettings, EmailStr, Field, PostgresDsn
+from sentry_sdk.integrations.django import DjangoIntegration
+
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+def as_bool(v: Optional[Union[str, List[str]]]):
+    if v is None:
+        return False
+
+    if isinstance(v, str):
+        v = [v]
+
+    return v[0].lower() in ("true", "yes", "t", "1")
+
+
+class Settings(BaseSettings):
+    """
+    Pydantic-powered settings, to provide consistent error messages, strong
+    typing, consistent prefixes, .venv support, etc.
+    """
+
+    #: The default database.
+    DATABASE_URL: PostgresDsn
```

Comment:
I would like the standard `PG*` environment variables to _also_ work, as they're meant to be common across software.